app/src/main/res/values/strings.xml

```diff
@@ -2586,9 +2586,8 @@ Please check the URL again."</string>
     <string name="code_shrinker_configure_shrinker">Configure R8/ProGuard by editing the rules file. This step is necessary because without a proper configuration, ProGuard might break your project.</string>
     <string name="code_shrinker_full_mode_def">Using this, you can configure specific local libs to be included in R8/ProGuarding process. (Note: Enabling this will ignore DEX files of the local libs you select and it will force Sketchware to compile them to dex, making compilation noticeably slower.)</string>
     <string name="code_shrinker_generate_debug_files">Generate debug files</string>
-    <string name="dependency_example">Example: org.ow2.asm:asm:9.5</string>
-    <string name="library_manager">Library Manager</string>
-    <string name="enter_dependency">Enter dependency</string>
-    <string name="dependency_info">You can find the dependency of a library in Github, typically in the README file.</string>
-    <string name="download">Download</string>
+    <string name="library_manager_dependency_example">Example: org.ow2.asm:asm:9.5</string>
+    <string name="library_manager">Local Library Manager</string>
```

Comment:
here

Reply:
here what? i'll repeat myself, no, it's still not `local_library_manager` and also not `Local library Manager` (≠ `Local Library Manager`). Local library Manager and Library Manager are also not the same, so `librara_manager` as prefix is wrong.

Reply:
check now